### DISCUSSION TITLE
I want to contribute to this project

### DISCUSSION THREAD

**Discussion by Kavan2003** [foss42/apidash#395] [Q&A] [answered]
I want to contribute to this project 
GirlScript Summer of Code
@Ankit Mahato
is project admin. 
Do I need to do any additional steps like filling form or any onboarding?
I have already registered for GirlScript Summer of Code.

**Comment by ashitaprasad** [comment, marked as answer]
Hi @Kavan2003,
There are no additional steps like filling form or any onboarding. But before selecting any issue that you want to work on, I will suggest you follow the steps given below:
- Go through the [README.md](https://github.com/foss42/apidash/blob/main/README.md) to understand what the project does.
- Go through the [CONTRIBUTING.md](https://github.com/foss42/apidash/blob/main/CONTRIBUTING.md) to setup API Dash local development environment and for all contribution guidelines.
- Take a look into the existing (or create new) project issues that you should now be able to understand better after getting familiar with API Dash.
- Choose an issue that you find interesting to work on and the comment on it. Feel free to reach out on our [Discord](https://discord.com/invite/2s49SCNfyJ) in case you have any doubts while working on the issue.
- After making code changes, send a PR for review linking the issue.
